cdap-ui/app/cdap/components/Replicator/Detail/SelectedTable/index.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useContext } from 'react';
+import withStyles, { WithStyles, StyleRules } from '@material-ui/core/styles/withStyles';
+import { DetailContext } from 'components/Replicator/Detail';
+import Heading, { HeadingTypes } from 'components/Heading';
+import If from 'components/If';
+
+const styles = (theme): StyleRules => {
+  return {
+    root: {
+      padding: '0 15px',
+      marginBottom: '25px',
+      display: 'flex',
+      alignItems: 'center',
+    },
+    reset: {
+      color: theme.palette.blue[100],
+      cursor: 'pointer',
+      marginLeft: '5px',
+
+      '&:hover': {
+        textDecoration: 'underline',
+      },
+    },
+    activeTableText: {
+      marginLeft: '10px',
+    },
+    heading: {
+      marginBottom: 0,
+    },
+  };
+};
+
+const SelectedTableView: React.FC<WithStyles<typeof styles>> = ({ classes }) => {
+  const { activeTable, setActiveTable } = useContext(DetailContext);
+
+  function handleReset() {
+    setActiveTable(null);
+  }
+
+  const heading = activeTable ? `"${activeTable}"` : 'All tables replicated';
```

Comment:
Nit: Why not just `const heading = activeTable || 'All tables replicated';`

Reply:
the design call out for the quotation mark for the name of a particular table

Reply:
Got it